### PR TITLE
V2: Metrics

### PR DESCRIPTION
**Pull request by fabriziovigevani**
**Preview**
Screen Shot 2020-10-15 at 1 36 39 PM

**When scrolling**
Screen Shot 2020-10-15 at 1 36 50 PM

### CODE REVIEW

File: 1 [hunk 1]
```diff
@@ -0,0 +1,72 @@
+Merge branch 'v2' into v2-missing-fixes
```
Comment (rperez89) [inline 1:1]: what is this file?

File: src/components/Feed/ProposalRankings.js [hunk 2]
```diff
@@ -22,24 +22,40 @@ function getRankingIcon(key, selected) {
...
-          onSelect={onChange}
-          selected={selected === index}
-        />
-      ))}
+      <div
+        css={`
+          display: flex;
+          align-items: center;
+          margin-bottom: ${1 * GU}px;
+
```
Comment (rperez89) [inline src/components/Feed/ProposalRankings.js:43]: lets remove this blank space

**Review by rperez89 — APPROVED** [review]
LGTM!

File: 1 (hunk 1 above)
Comment (fabriziovigevani) [reply 1:1]: Ooops 😅

**Pull request merged**